File: complete/src/BookManager.Infrastructure/Book.cs
namespace BookManager.Infrastructure
{
    public class Book
    {
        public string Title { get; set; }

        public string AuthorFirstName { get; set; }
        public string AuthorLastName { get; set; }

        public int YearPublished { get; set; }
        public string Id { get; set; }
    }
}

File: lab_01/src/bookmanager.ui/src/book.js
import React from 'react';
import { Button } from 'react-bootstrap';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faTrash } from '@fortawesome/free-solid-svg-icons';

function Book(props) {
  const {
    title,
    authorFirstName,
    authorLastName,
    yearPublished,
    id,
    handleBookDeleteClick,
  } = props;

  return (
    <tr>
      <td>{title}</td>
      <td>{authorFirstName} {authorLastName}</td>
      <td>{yearPublished}</td>
      <td>
        <Button id="delete-book-button" data-id={id} variant="danger" onClick={handleBookDeleteClick}>
          <FontAwesomeIcon icon={faTrash} />
        </Button>
      </td>
    </tr>
  );
}

export default Book;
